templates/sag-cc-repos-ga/README.md

```diff
@@ -18,10 +18,12 @@
 -->
 # Command Central Internal GA Product and Fix Repositories
 
-Use this template to create internal GA Software AG product and fix repositories in Command Central that replicate the production repositories, provided to Software AG customers.
+Only for internal use. With this template, you can add the General Availability (GA) or release-candidate sandboxes, located on the Software AG internal install servers, as product and fix repositories in Command Central.
 
 ## Requirements
 
+None.
```

Comment:
Elly, the same as previous: credentials have to be created.
In this case they are done by sag-cc-creds-dev template.

Reply:
Thanks, updated!